### PR TITLE
[TS] LPS-121757

### PR DESCRIPTION
**Pull request by bakayattila**
hi Team,

Could you please review this pull request?

https://issues.liferay.com/browse/LPS-121757
Best,
Attila

### CODE REVIEW

**Comment by bakayattila** [comment]
ci:test:sf

**Comment by bakayattila** [comment]
ci:test:relevant

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java [hunk 1]
```diff
@@ -238,7 +238,9 @@ public int getAbstractLength() {
 
 		String selectionStyle = getSelectionStyle();
 
-		if (selectionStyle.equals("dynamic")) {
+		if (selectionStyle.equals("dynamic") ||
+			selectionStyle.equals("asset-list")) {
```
Comment (pavel-savinov) [inline modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java:242]: @bakayattila Tag names for the asset lists must be from the corresponding asset list entry properties, not from the portlet properties.

**Comment by pavel-savinov** [comment]
@bakayattila Could you please check my commit and resend? Thanks!

**Comment by pavel-savinov** [comment]
ci:close

**Pull request closed without merging**